mep/people/tests/test_views.py

```diff
@@ -1059,7 +1060,8 @@ def test_member_map(self):
         response = self.client.get(url)
         assert response.status_code == 200
         # library address is rendered as "null"
-        self.assertContains(response, "libraryAddress = null")
+        self.assertContains(response,
+            '<script id="library-address" type="application/json">null</script>')
```

Comment:
whoa, these have `type="applilcation/json"`? that's pretty cool. we may not actually _need_ the nonce anymore, since in this version the browser won't try to parse the script automatically, iirc